packages/wallet-sdk/src/provider/MobileRelayUI.ts

```diff
@@ -41,12 +41,24 @@ export class MobileRelayUI implements WalletUI {
   }
 
   private useLocationMethod = false;
-  setUseLocationMethod(useLocationMethod: boolean): void {
+  private useCustomScheme = false;
+
+  setDeepLinkConfig({
+    useCustomScheme = false,
+    useLocationMethod = false,
+  }: {
+    useLocationMethod?: boolean;
+    useCustomScheme?: boolean;
+  }): void {
     this.useLocationMethod = useLocationMethod;
+    this.useCustomScheme = useCustomScheme;
   }
 
   private redirectToCoinbaseWallet(walletLinkUrl?: string): void {
-    const url = new URL("https://go.cb-w.com/walletlink");
+    const baseUrl = this.useCustomScheme
+      ? "cbwallet://"
+      : "https://go.cb-w.com/";
+    const url = new URL("walletlink", baseUrl);
```

Comment:
The url opened seems to be `cbwallet:///walletlink?redirect_url...`. Note the three `///`

Reply:
thanks. what would be the optimal url for cbwallet walletlink path?
would it make the logic simpler if we have it the same format like `cbwallet://go.cb-w.com/walletlink`